NuRadioMC/SignalGen/emitter.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import h5py
+from scipy.interpolate import interp1d
+from NuRadioReco.utilities import units, fft
+import logging
+logger = logging.getLogger("SignalGen.emitter")
+import os
+
+
+def get_time_trace(amplitude, N, dt, model, full_output=False, **kwargs):
+    """
+    returns the voltage trace of an emitter
+
+    We implement only the time-domain solution and obtain the frequency spectrum
+    via FFT (with the standard normalization of NuRadioMC). This approach assures
+    that the units are interpreted correctly. In the time domain, the amplitudes
+    are well defined and not details about fourier transform normalizations needs
+    to be known by the user.
+
+    Parameters
+    ----------
+    amplitude : float 
+        strength of a pulse
+    N : int
+        number of samples in the time domain
+    dt: float
+        time bin width, i.e. the inverse of the sampling rate
+    model: string
+        specifies the signal model
+        * delta_pulse: a simple signal model of a delta pulse emitter
+        * cw : a sinusoidal wave of given frequency
+        * square : a rectangular pulse of given amplituede and width
+        * tone_burst : a short sine wave pulse of given frequency and desired width
+        * idl & hvsp2 : these are the waveforms generated in KU lab and stored in hdf5 files
+        * gaussian : represents a gaussian pulse where sigma is defined through the half width at half maximum
```

Comment:
Hi, it's very minor, but to prevent errors in generating the documentation (#350 ), could you please add empty lines around the lists in your docstrings?
I.e.
```
       specifies the signal model

        * delta_pulse: a simple signal model of a delta pulse emitter
        * cw : a sinusoidal wave of given frequency
        * square : a rectangular pulse of given amplituede and width
        * tone_burst : a short sine wave pulse of given frequency and desired width
        * idl & hvsp2 : these are the waveforms generated in KU lab and stored in hdf5 files
        * gaussian : represents a gaussian pulse where sigma is defined through the half width at half maximum

      full_output: bool (default False)
```